runtime/worker.rs

```diff
@@ -189,6 +192,9 @@ pub struct WorkerOptions {
   pub compiled_wasm_module_store: Option<CompiledWasmModuleStore>,
   pub stdio: Stdio,
   pub feature_checker: Arc<FeatureChecker>,
+
+  /// V8 code cache for module and script source code.
+  pub code_cache: Option<Arc<dyn CodeCache>>,
```

Comment:
Nitpick, could you rename it to `v8_code_cache`? I think it will be more helpful later when coming back to this code after some time, just `code_cache` feels generic and I would think it's one of our internal caches that doesn't interact with V8